app/javascript/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import greetingReducer from './reducers/greetingReducer';
+
+const store = createStore(greetingReducer, applyMiddleware(thunk));
+
+export default store;
```

Comment:
- [ OPTIONAL ] I would suggest you to update redux with redux/toolkit is easier to manage
![image](https://user-images.githubusercontent.com/62705947/221903414-47d5266f-8f89-4c3f-93ae-9c9db7b0e25a.png)

Reply:
how?